### ISSUE TITLE
Training Request Forms

### ISSUE THREAD

**Issue by samozzy** [newtheatre/nt-training#18]
The Training Site currently records training given, not competence, even though these two are linked. Zoe has suggested an architectural revamp to honour this further.

## 1) Training Requests
Users are able to request training (`/training/request/new`) using a form not dissimilar to `/training/new`. This will then generate a list of requested sessions at `/training/request` which can then be actioned and organised.
Training requests will be able to be marked as completed via a button which transfers them into a training session.

The below is nonsense but kept for context.
<details>
## 2) Training Given
Training sessions become the first step in a process. Once a training session has been given, trainees' records are then updated in amber (rather than the current green) to reflect this. 

## 3) Competence Agreed (or a good name)
Users are then invited (via email?) to reflect on this training session and their confidence in performing these skills. If they feel confident they can complete the training item; otherwise, they can request further training/practice in this skill. This can be selected on an item-by-item basis. 
**Competence Agreed**: Users then receive a green training item on their record, indicating their competence in this skill.
**Competence not agreed**: Users retain the amber item, and a new training request is created for those skills that want some more work.

Decision needed: Competence agreed can only be completed by the trainee, and not by the trainer.

Note: Competence can be revoked by adding a training session more recently than competence agreed. 
</details>

**Comment by samozzy** [comment]
# Reflections, 18 months on...

## Training Requests

Is a good idea. Needs working out on the how. Remember that members can just ask the relevant person and all the contacts are on the pages. It's easy if that person is approachable and not a dick, and usually 'ability to be a good trainer' correlates with that.

Turning an online request into a training session is not a dumb idea as a quick and easy button.

## Competence assessed 

Is basically bollocks and against the premise of this whole programme.
Competence should be agreed by the trainee at the point of training, and a good session will include the opportunity to practice the skills learned.
**Non-safety critical items** are trained and ticked. The training site records training given. No problem. Asses covered.
**Safety-critical items** are inherently only signed off when individuals have proven their ability to complete the task at hand.

Anyone observed as incompetent on a safety-critical task should be retrained at the earliest convenience and be informed that they aren't doing the thing safely. Anyone who has received training in a safety-critical task will have been informed along the way to know their limits and not do the thing if they don't think they can do it safely.

# TL;DR:

* If the person isn't trained right in the first place, they aren't going to be competent at the task.
* Trainers should be approachable and easy to talk to, which will lead to higher quality training sessions.
* Competency is assessed within the current model of training sessions.

# Moving forward

The scope of this issue is now *exclusively* for training request forms, which is considered a *low-priority* issue given a) the other issues outstanding and b) that people should be able to talk to each other like humans.
